widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -0,0 +1,194 @@
+import type { CustomColorsTypes } from './StyleLayout.types';
+import type { Mode } from '../../store/config';
+import type { WidgetColorsKeys } from '@rango-dev/widget-embedded';
+
+import {
+  ChevronDownIcon,
+  ChevronUpIcon,
+  Collapsible,
+  CustomColorsIcon,
+  Divider,
+  Typography,
+} from '@rango-dev/ui';
+import React, { useEffect, useState } from 'react';
+
+import { ColorItemPicker } from '../../components/ColorItemPicker';
+import { WIDGET_COLORS } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { getMainColor } from '../../utils/colors';
+
+import {
+  ColoredCircle,
+  ColorsContent,
+  CustomColorCollapsible,
+  CustomColors,
+  FieldTitle,
+  Row,
+} from './StyleLayout.styles';
+
+const Colors = (props: { mainColor?: string; secondColor?: string }) => {
+  return (
+    <ColorsContent>
+      <ColoredCircle style={{ backgroundColor: props.mainColor }} />
+      {props.secondColor !== undefined && (
+        <ColoredCircle
+          position="absolute"
+          style={{
+            backgroundColor: props.secondColor,
+            right: -10,
+          }}
+        />
+      )}
+    </ColorsContent>
+  );
+};
+
```

Comment:
We have a color issue here.

![Screenshot from 2023-10-30 16-03-54](https://github.com/rango-exchange/rango-client/assets/99180934/fe348e8d-6214-48cc-be74-3ca39a96ab32)